### PR TITLE
CAPI-1210 : Add platform services to catalog during installation

### PR DESCRIPTION
**Pull request by SiniWilson**

### CODE REVIEW

**Comment by suryajak** [comment]
Can you resolve merge conflicts in your PR?

**Comment by SiniWilson** [comment]
@suryajak  Resolved the merge conflict

**Comment by suryajak** [comment]
@SiniWilson: Just as you did in core, please raise this PR against v1.3.1 branch.

**Comment by SiniWilson** [comment]
Changed the base branch to v1.3.1

File: installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh [hunk 1]
```diff
@@ -119,14 +50,31 @@ sed -i "s/{inst_stack_prefix}/$stackprefix/g" ./jazz-core/serverless-config-pack
...
+          "SERVICE_STATUS":{"S":"active"},
+          "SERVICE_METADATA":{"M":{
+                          "securityGroupIds":{"S":"'$securityGroupIds'"},
+                          "subnetIds":{"S":"'$subnetIds'"},
+                          "iamRoleARN":{"S":"'$iamRoleARN'"},
+                          "providerMemorySize":{"S":"256"},
+                          "providerRuntime":{"S":"nodejs4.3"},
+                          "providerTimeout":{"S":"160"},
+                          "runtime":{"S":"nodejs"},
+                          "service":{"S":"'$element'"},
```
Comment (suryajak) [inline installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh]: service is not part of meta-data, its already part of the name.

File: installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh [hunk 2]
```diff
@@ -119,14 +50,31 @@ sed -i "s/{inst_stack_prefix}/$stackprefix/g" ./jazz-core/serverless-config-pack
...
+          "SERVICE_DOMAIN":{"S":"platform"},
+          "SERVICE_NAME":{"S":"'$element'"},
+          "SERVICE_RUNTIME":{"S":"nodejs"},
+          "SERVICE_STATUS":{"S":"active"},
+          "SERVICE_METADATA":{"M":{
+                          "securityGroupIds":{"S":"'$securityGroupIds'"},
+                          "subnetIds":{"S":"'$subnetIds'"},
+                          "iamRoleARN":{"S":"'$iamRoleARN'"},
+                          "providerMemorySize":{"S":"256"},
+                          "providerRuntime":{"S":"nodejs4.3"},
```
Comment (suryajak) [inline installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh]: email service requires runtime as nodejs6.10, so we need a way to handle this.

File: installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh (hunk 1 above)
Comment (SiniWilson) [reply installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh]: Removed the service from metadata

File: installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh (hunk 2 above)
Comment (SiniWilson) [reply installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh]: Changed to 6.10

**Pull request merged**